include/sdf_tools/sdf.hpp

```diff
@@ -136,6 +136,13 @@ namespace sdf_tools
 
         std::pair<double, bool> EstimateDistance4d(const Eigen::Vector4d& location) const;
 
+        // Estimate the distance between the given point and the outer boundary of the SDF
+        std::pair<double, bool> DistanceToBoundary(const double x, const double y, const double z) const;
```

Comment:
std::pair<double, bool>
Presumably double is the distance.
bool is "is inside"? "is outside"?
Document, or make std::pair<double, bool> a separate type (with good naming)

Reply:
I'm not entirely sure where to document this given that it is the return type of something like 20 different functions. FWIW, the bool reports if the given location is inside the SDF.

Reply:
How about we push this off to be fixed when #10 addressed, and resolve the whole set there?

Reply:
Not my repo, I'm not going to stop you from merging due to this.